modules/ROOT/pages/autocompleter.adoc

```diff
@@ -23,7 +23,12 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` (of one character) |Required |The character to trigger the autocompleter.
+include::partial$misc/admon-deprecated-6.2v.adoc[]
+It has been replaced with the `+trigger+` option, which should be used instead.
+|trigger |`+string+` |Required |The string to trigger the autocompleter.
+include::partial$misc/admon-requires-6.2v.adoc[]
+It replaces the deprecated `+ch+` option, and should be used instead of that option.
```

Comment:
```suggestion
include::partial$misc/admon-requires-6.2v.adoc[]
```
You really don't need to be repeating yourself twice here.

Reply:
I thought that was what you were asking in the other thread?